### ISSUE TITLE
Better UI

### ISSUE THREAD

**Issue by derfenix** [derfenix/webarchive#1]
Current UI was made in one evening by a backend developer. It is just works, but ugly and may have better UX.

**Comment by TheRouce** [comment]
<details>
<summary>WIP ui using mostly tailwindcss</summary>
<br>

<img src="https://github.com/derfenix/webarchive/assets/40551268/02cd230f-81ea-46b8-9e10-5be55038ee67" alt="Image 1">
<img src="https://github.com/derfenix/webarchive/assets/40551268/f661ea05-de5a-4e24-b499-88a48bcf5226" alt="Image 2">

</details>

Next I'm trying to sort out what would be good new ways to display the archived sites, obviously a grid would be a good place to start but if webarchive also grabbed a screenshot then it could display a preview image or something. Anyways, let me know what you think.

**Comment by derfenix** [comment]
Hello
  
I think it is better to fit into current service abilities and support new functionality only after it will be implemented, not trying to guess what will be implemented next. I still have no static roadmap and have no idea what and when will be implemented. My point now is to fix all bugs and make existing functionality stable enough.
  
Also, I hope you saw the theming support exists here. So you are free to add any number of themes alongside with existing `basic` theme and switch it via environment variable. Just try to keep it small enough (keep it free from megabytes of scripts, I mean).

**Comment by TheRouce** [comment]
Yeah it's pretty small still and I'm working on a couple variations so I'll see about adding the functionality to switch themes
